src/uikit.cs

```diff
@@ -10220,12 +10220,21 @@ interface UISearchContainerViewController
 	
 	[iOS (8,0)]
 	[BaseType (typeof (UIViewController))]
-	partial interface UISearchController : UIViewControllerTransitioningDelegate, UIViewControllerAnimatedTransitioning {
+	[DisableDefaultCtor] // designated
+	partial interface UISearchController : UIViewControllerTransitioningDelegate, UIViewControllerAnimatedTransitioning
+	{
+		[Export ("init")]
+		[DesignatedInitializer]
```

Comment:
I think this one shouldn't be `[DesignatedInitializer]`
```suggestion

```

According to Apple's header files:

```
Headers/UISearchController.h
39:- (instancetype)initWithSearchResultsController:(nullable UIViewController *)searchResultsController NS_DESIGNATED_INITIALIZER;
```

Reply:
Agreed. It's not in the headers nor in the documentation
https://developer.apple.com/documentation/uikit/uisearchcontroller?language=objc